### ISSUE TITLE
Hive `eth` transaction test fails

### ISSUE THREAD

**Issue by Rjected** [paradigmxyz/reth#2706]
### Describe the bug

Currently the hive p2p test `TestTransaction` fails:
https://github.com/paradigmxyz/reth/actions/runs/4986338461/jobs/8927244329#step:6:38

```
skipping suite "discv4" because it doesn't match test pattern eth/TestTransaction
1..1
not ok 1 TestTransaction
# Testing tx propagation 0: sending tx 0x35ac5799d8bd35fcfcd474015abca116d808d6da7f4afc6a5f613791199e6fba 1000000 21000
# send successful tx test failed: unexpected message in sendSuccessfulTx: (*ethtest.Error)(could not read from connection: read tcp 172.17.0.3:43788->172.17.0.4:30303: i/o timeout)
# 
exit status 1
skipping suite "snap" because it doesn't match test pattern eth/TestTransaction
```

I don't yet have any insight into why this test fails.

### Steps to reproduce

Run hive:
```
hive --sim "devp2p$" --sim.limit "eth/TestTransaction" --client reth
```

### Code of Conduct

- [X] I agree to follow the Code of Conduct

**Comment by mattsse** [comment]
this is likely the same issue as the `syncing`

perhaps we should insert txs if we're syncing?

**Comment by Rjected** [comment]
Fixed by #2722
